JuliaInterface/tst/utils.tst

```diff
@@ -100,5 +100,13 @@ Error, JuliaGetFieldOfObject: <field_name> must be a string
 gap> JuliaGetFieldOfObject(JuliaModule("Base"), "not-a-field");
 Error, type Module has no field not-a-field
 
+##
+gap> NameFunction(Julia.Base.parse);
+"parse"
+gap> Display(Julia.Base.parse);
+function ( arg... )
+    <<kernel code>> from Julia:parse
```

Comment:
The `<<kernel code>> from ` bit is fixed by GAP (at least currently), and so is the "FOO:BAR" format for the final part. Nice would of course be something like: `<<Julia function Base.parse>>`. Alas, I also don't really know how to produce `Base.parse` instead of `parse`. Perhaps it's as simply as checking for the parent module, getting its name, and prepending it (with a dot), and then repeating this until the top level is reached? Alas, we can figure that out later.

Reply:
I would say it is fine that way for now. If people start to complain we can still fix it. A sensible top-level would be `Main` I guess. Or maybe `Top`?